pkg/apply/processor/create.go

```diff
@@ -79,6 +81,21 @@ func (c *CreateProcessor) Check(cluster *v2.Cluster) error {
 	if err != nil {
 		return err
 	}
+	ociList, err := c.ImageManager.Inspect(cluster.Spec.Image...)
+	if err != nil {
+		return err
+	}
+	imageTypes := sets.NewString()
+	for _, oci := range ociList {
+		if oci.Config.Labels != nil {
+			imageTypes.Insert(oci.Config.Labels[contants.ImageTypeKey])
+		} else {
+			imageTypes.Insert(string(v2.AppImage))
+		}
+	}
+	if !imageTypes.Has(string(v2.RootfsImage)) {
+		return errors.New("not support only run not rootfs images in this cluster at create processor")
```

Comment:
can't apply ApplicationImage, kubernetes cluster not found,  need to run a BaseImage